### DISCUSSION TITLE
RER preset shows up every single encounter

### DISCUSSION THREAD

**Discussion by Kingzzor** [Aelto/W3_RandomEncounters_Tweaks#227] [General]
Everytime I encounter an enemy RER preset shows up, is there a way I can save the settings so they dont show up every time?  It messes my game up every time it shows up. 

![436788716_397099033094115_8816406345671637253_n](https://github.com/Aelto/tw3-random-encounters-reworked/assets/166779585/b6f428fa-4106-4845-a328-3927802143c1)

**Comment by Aelto** [comment]
Closed for duplicate, answered elsewhere.

Possible cause:
- missing menu filelists, a section in the in-depth guide explains how to update them if needed
